app/routes/invitations.rb

```diff
@@ -0,0 +1,76 @@
+module ExercismWeb
+  module Routes
+    class Invitations < Core
+      post '/teams/:slug/invitation' do |slug|
+        please_login
+        only_for_team_managers(slug, "You are not allowed to add team members.") do |team|
+          team.invite_with_usernames(params[:usernames], current_user)
+          team.save
+
+          flash[:success] = "Invitation sent to team #{slug}"
+          redirect "/teams/#{slug}/manage"
+        end
+      end
+
+      post '/teams/:slug/invitation/accept' do |slug|
+        please_login
+        only_with_existing_team(slug) do |team|
+          invite = invite_for_user_and_team(current_user, team)
+
+          unless invite.present?
+            flash[:error] = "You don't have a pending invitation to this team."
+            redirect "/"
+          end
+
+          invite.accept!
+          flash[:success] = "You are now a member of team #{slug}."
+          redirect "/teams/#{slug}"
+        end
+      end
+
+      post '/teams/:slug/invitation/refuse' do |slug|
```

Comment:
Ditto on `invitations` (and still uneasy about `refuse`, but here, too, let's just keep it for now).